conans/client/conf/config_installer.py

```diff
@@ -207,6 +211,44 @@ def _save_configs(configs_file, configs):
                                   indent=True))
 
 
+def _generate_sched_file(cache):
+    now = str(datetime.utcnow())
```

Comment:
```
Warning Because naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. As such, the recommended way to create an object representing the current time in UTC is by calling datetime.now(timezone.utc).
```
I am also not sure about this one, as it's not monotonic (e.g. affected by system clock changes)

Reply:
done